core/Rename.cpp

```diff
@@ -137,15 +216,36 @@ namespace olympia
 
                 // TODO: Register renaming for sources
                 const auto & srcs = renaming_inst->getSourceOpInfoList();
+                if(srcs.size() > 1){
+                    int i = 0;
+                    i++;
+                }
```

Comment:
This is a weird `if` statement.  😄

Reply:
Sorry some old debugging code 😅. Removed in [3eb203c](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/3eb203c761cd11fbebea435284f6d14ba45437a7).